discovery-infra/test_infra/controllers/node_controllers/host.py

```diff
@@ -63,6 +63,17 @@ def kill_service(self, service):
         logging.info("Killing service %s on host %s", service, self.name)
         self.run_command(f'sudo systemctl kill {service}.service || true')
 
+    def kill_installer(self):
+        current_proc = self.run_command("sudo su root -c 'podman ps | grep assisted-installer'")
+        logging.info(f"Current running installer container details on {self.name}: {current_proc}")
+        container_id = self.run_command("sudo su root -c '(podman ps | grep assisted-installer | cut -d \" \" -f1)'")
+        self.kill_podman_container_by_id(container_id.strip())
+
+    def kill_podman_container_by_id(self, container_id):
+        logging.info(f"Killing container: {container_id}")
+        output = self.run_command(f"sudo su root -c 'podman kill {container_id}'")
+        logging.info(f"Output of Kill installer: {output}")
```

Comment:
"Output of Kill Container", not installer

Reply:
the idea was to use this func in kill_installer ? why need them both